src/app/CommandInputField.tsx

```diff
@@ -85,6 +93,16 @@ export const CommandInputField = observer((props: { controller: BottomPanelContr
       const selected = controller.selected;
       const autoCompletionValues = controller.autoCompletionValues;
 
+      if (selected === null && autoCompletionValues.length <= 0 && lastCmd.length > 0) {
+        if (lastCmdIndex.get() + 1 < lastCmd.length) {
+          lastCmdIndex.set(lastCmdIndex.get() + 1);
+          input.value = lastCmd[lastCmdIndex.get()];
+        } else {
+          input.value = "";
+          lastCmdIndex.set(lastCmd.length);
+        }
```

Comment:
Also, set the carpet position to the end of the command line after using the hotkey.

Reply:
Updated, thank you.